### PR TITLE
Distance Calculating App 

### LINKED ISSUE #236: Distance calculating app.
**Is your feature request related to a problem? Please describe.**
An app that can show the distance between your current position(located using user's current IP) and user searched position on Map.

**Additional context**
This app will have a frontend too, to show searched position on the map.

### PR DESCRIPTION
**Pull request by pritamp17**
Distance Calculating Django app

Fixes #236 

### Checklist:
- [x] My branch is up-to-date with upstream/master branch.

### Snapshot:
django-2-test2
django-1-test2
django-3-test2

### CODE REVIEW

**Comment by pritamp17** [comment]
@Tejas1510 please do a review of this PR.

File: Python/Calculate-distance/Readme.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+2. GeoLite2-country.mmdb
+to geoip folder
+```
+```bash
+pip install -r requirements.txt
+```
+```bash
+python manage.py runserver
+```
+# And there you go
```
Comment (atarax665) [inline Python/Calculate-distance/Readme.md:44]:
```suggestion
# And here you go
```

File: Python/Calculate-distance/Readme.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+![](https://github.com/pritamp17/Calculate-Distance/blob/main/django-1-test2.png)
+![](https://github.com/pritamp17/Calculate-Distance/blob/main/django-2-test2.png)
+![](https://github.com/pritamp17/Calculate-Distance/blob/main/django-3-test2.png)
+
+# Authur
```
Comment (atarax665) [inline Python/Calculate-distance/Readme.md:49]:
```suggestion
# Author
```

File: Python/Calculate-distance/Readme.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
+# Distance Calculating Django App
+App that can show distance between user's location and searched location on map.
+
+# Packages Used
```
Comment (atarax665) [inline Python/Calculate-distance/Readme.md:4]: Please also mention which version of Django this project is compatible with.  Also add it in requirements.txt.

**Review by atarax665 — CHANGES_REQUESTED** [review]

File: Python/Calculate-distance/Readme.md (hunk 3 above)
Comment (pritamp17) [reply Python/Calculate-distance/Readme.md:4]: @atarax665 I have made the changes, do I need to make any other changes?

File: Python/Calculate-distance/Readme.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+cd Hacking-Scripts/python/Calculate-distance
+```
+```
+create folder named 'geoip' in main project directory.
+```
+```
+download city and country database
+```
+[For downloading city and country database ](https://www.maxmind.com/en/accounts/497315/people/84e6213c-91a4-4e02-ae2e-1d709084c544)
+1. ![](https://github.com/pritamp17/Calculate-Distance/blob/main/1.paste.png)
```
Comment (atarax665) [inline Python/Calculate-distance/Readme.md:29]:
```suggestion
1. ![](1.paste.png)
```

File: Python/Calculate-distance/Readme.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+2. ![](https://github.com/pritamp17/Calculate-Distance/blob/main/2.paste.png)
```
Comment (atarax665) [inline Python/Calculate-distance/Readme.md:30]:
```suggestion
2. ![](2.paste.png)
```

File: Python/Calculate-distance/Readme.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+to geoip folder
+```
+```bash
+pip install -r requirements.txt
+```
+```bash
+python manage.py runserver
+```
+# And here you go
+![](https://github.com/pritamp17/Calculate-Distance/blob/main/django-1-test2.png)
```
Comment (atarax665) [inline Python/Calculate-distance/Readme.md:45]:
```suggestion
![](django-1-test2.png)
```

File: Python/Calculate-distance/Readme.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+![](https://github.com/pritamp17/Calculate-Distance/blob/main/django-2-test2.png)
```
Comment (atarax665) [inline Python/Calculate-distance/Readme.md:46]:
```suggestion
![](django-2-test2.png)
```

File: Python/Calculate-distance/Readme.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+![](https://github.com/pritamp17/Calculate-Distance/blob/main/django-3-test2.png)
```
Comment (atarax665) [inline Python/Calculate-distance/Readme.md:47]:
```suggestion
![](django-3-test2.png)
```

**Review by atarax665 — CHANGES_REQUESTED** [review]

File: Python/Calculate-distance/Readme.md (hunk 8 above)
Comment (pritamp17) [reply Python/Calculate-distance/Readme.md]: @atarax665 I  have re-uploaded images using https://imgur.com/a/m9urkX4.

**Pull request merged**